{"name": "test_fixture_scope_function01", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 > 100", "trace": "random_num = 0\n\n    @allure.feature(\"test_02_fixture\")\n    @allure.story(\"scope\")\n    def test_fixture_scope_function01(random_num):\n        print(f'random_num: {random_num}')\n>       assert random_num > 100\nE       assert 0 > 100\n\ntest_02_fixture_for_allure.py:121: AssertionError"}, "start": 1627037549868, "stop": 1627037549868, "uuid": "47e0a006-35c2-405c-a6c7-3b579cc4b885", "historyId": "541b59f09b5a50b681f45afbfa986c33", "testCaseId": "e06d2c0c803f299dd3fb695682f833cf", "fullName": "test_02_fixture_for_allure#test_fixture_scope_function01", "labels": [{"name": "story", "value": "scope"}, {"name": "feature", "value": "test_02_fixture"}, {"name": "suite", "value": "test_02_fixture_for_allure"}, {"name": "host", "value": "JULIZHOU-6JV5P"}, {"name": "thread", "value": "4280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_02_fixture_for_allure"}]}